config/util.go

```diff
@@ -96,6 +96,11 @@ type DurationOpt struct {
 // ParseDurations takes a time.Duration src and saves it to the given dst.
 func ParseDurations(component string, args ...*DurationOpt) error {
 	for _, arg := range args {
+		if arg.Duration == "" {
```

Comment:
Any duration field if not passed will take the default value, right?

Reply:
yes, since all configs are defaulted before being parsed.